src/views/v2/rsvp/ari/sidebar/guest-list.php

```diff
@@ -47,7 +47,7 @@
 	<li class="tribe-tickets__rsvp-ar-guest-list-item">
 		<button>
```

Comment:
does this button need to be disabled?

Reply:
No, the idea is that the button will "switch" the form to present the other guest form. It's more a styling thing. Although this is just for presentation purposes now. This will be handled via JavaScript (addition/deletion of guests).